docs/examples/plot_backtracking_sloped_terrain.py

```diff
@@ -0,0 +1,176 @@
+"""
+Backtracking on sloped terrain
+==============================
+
+Modeling backtracking for single-axis tracker arrays on sloped terrain.
+"""
+
+# %%
+# Tracker systems use backtracking to avoid row-to-row shading when the
+# sun is low in the sky. The backtracking strategy orients the modules exactly
+# on the boundary between shaded and unshaded so that the modules are oriented
+# as much towards the sun as possible while still remaining unshaded.
+# Unlike the truetracking calculation (which only depends on solar position),
```

Comment:
Is "truetracking" (one word) widely accepted? I'd probably write true-tracking even at the cost of inconsistency with backtracking.

Reply:
The existing single-axis tracker example uses the hyphenated version.